k8s/backend-configmap.yaml

```
@@ -0,0 +1,80 @@
+        "declaration": true,
+        "declarationMap": true,
+        "sourceMap": true
+      },
+      "include": ["src/**/*"],
+      "exclude": ["node_modules", "dist"]
+    }
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: maas-backend-source
+  namespace: llm
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)